aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
> That was NEVER the intention of the DN property

I'm not saying we intended scripts or even humans to do that. I'm saying that giving the whole side-effects picture is better for debugability, which is the intention of the DN(s) attribute exposed by Neutron (as far as I recall from our discussions, which might be wrong/changed).

> Why? One of the reasons we have AIM is to hide these very ugly details from the AIM consumer

We do hide them. The DN(s) attribute is not for the northbound project to understand.

> Actually, I think it should be the other way around

I was under the impression that we wanted to separate ACI specifics from the AIM library and delegate them to the AID module (when we discussed moving "uglyness" in the agent). If that's not the case anymore, then it's a better solution to decorate the AIM resources.